test/resizing.jl

```diff
@@ -144,5 +144,19 @@ end
             @test ndims(R) == 0
             @test !(R === A)
         end
+
+    @testset "Interpolation" begin
+        img = rand(16,16)
+        out = ImageTransformations.imresize(img,(128,128),Lanczos4OpenCV())
+        @test size(out) == (128,128)
```

Comment:
> Is it easy to see what test hits line 290 in the last commit? It would help me design a test to hit 305. I was hoping this might be in the detailed codecov report I cannot see.

Also need to test two more cases:

- when `img` has offset axes: `img = OffsetArray(rand(16, 16), -1, -1)` (a zero-indexed array) -- this is where the if-branch in 305 hits.
- when `sizes` are axes: `imresize(img, (0:127, 0:127), Lanczos4OpenCV())` -- this is the `Indices` method.